frontend/views/containers/modals/CreateGroup.vue

```diff
@@ -1,48 +1,54 @@
 <template lang='pug'>
-//- TODO: move this into modal
-main.main#create-group-page
-  .steps
-    router-link.step(
+modal-base-template
+  .steps(v-if='currentStep + 1 < config.steps.length')
+    button.step(
       v-for='(step, index) in config.steps'
       :key='index'
-      :to='{name: step}'
       :class='[currentStep === index ? "active" : "", currentStep < index ? "next" : ""]'
+      @click='redirect(step)'
+      v-if='step != "GroupWelcome"'
     ) {{ index + 1 }}
 
+  .wrapper.mobile-steps.subtitle
+    i18n(tag='span') Step
+    |  {{ currentStep }}
+    i18n(tag='span')  of
+    |  {{ config.steps.length + 1 }}
```

Comment:
Improper i18n usage, please see other examples in the codebase for how to do this correctly

Reply:
Can you link me to an exemple please?

Reply:
Search the project for `:args=`. For example, here: https://github.com/okTurtles/group-income-simple/blob/a6fe30d75e4b529a49e1b2dba04f51e61a5acd21/frontend/views/components/Proposals/ProposalBox.vue#L7-L11